localisation/00_zuwc_united_localisation.csv

```diff
@@ -30,11 +30,11 @@ EVTDESC_WCCLS_100;You have completed your magical training by performing excepti
-EVTDESC_WCCLS_500;"The Light speaks to you very clearly now�; its powers of healing and preservation are a second language to you. You have left marks on those you have helped and protected with your divine words. The forces of darkness shy away from that which is a shining beacon of the Light: you. The pains and nightmares caused by evil find themselves utterly purged by your touch. May Azeroth see you develop into the greatest priest ever known.";� pr�sent, vous comprenez clairement la Lumi�re; ses pouvoirs salvateurs n'ont plus aucun secret pour vous. Vos paroles divines ont marqu�s tous ceux que vous avez aid�s et prot�g�s alors que les Forces des T�n�bres craignent le phare �tincelant de lumi�re que vous �tes devenu. Les maux caus�s par les Forces du Mal disparaissent totalement � votre contact. Puisse Azeroth vous voir devenir le plus grand pr�tre de tous les temps.;Ihr habt Eure Ausbildung zum Priester abgeschlossen und habt die Schulen der Heilung und der Dunkelheit studiert. Es liegt an Euch, das Geschenk des heiligen Lichts richtig anzuwenden. Seit dem Aufkommen des Lichts haben Priester ihr Volk unterst�tzt um selbst in den dunkelsten Zeiten einen Funken Hoffnung zu verbreiten. Viel Gl�ck, Priester[From.GetinFemale]! Verzerrt das heilige Licht nicht umsonst - es kann ihren Tr�ger in einen Abgrund aus Dunkelheit und Verzweiflung st�rzen lassen.;;;;;;;;;x
+EVTDESC_WCCLS_500;The Light speaks to you very clearly now. Its powers of healing and preservation are a second language to you. You have left marks on those you have helped and protected with your divine words. The forces of darkness shy away from that which is a shining beacon of the Light: you. The pains and nightmares caused by evil find themselves utterly purged by your touch. May Azeroth see you develop into the greatest priest ever known.;� pr�sent, vous comprenez clairement la Lumi�re; ses pouvoirs salvateurs n'ont plus aucun secret pour vous. Vos paroles divines ont marqu�s tous ceux que vous avez aid�s et prot�g�s alors que les Forces des T�n�bres craignent le phare �tincelant de lumi�re que vous �tes devenu. Les maux caus�s par les Forces du Mal disparaissent totalement � votre contact. Puisse Azeroth vous voir devenir le plus grand pr�tre de tous les temps.;Ihr habt Eure Ausbildung zum Priester abgeschlossen und habt die Schulen der Heilung und der Dunkelheit studiert. Es liegt an Euch, das Geschenk des heiligen Lichts richtig anzuwenden. Seit dem Aufkommen des Lichts haben Priester ihr Volk unterst�tzt um selbst in den dunkelsten Zeiten einen Funken Hoffnung zu verbreiten. Viel Gl�ck, Priester[From.GetinFemale]! Verzerrt das heilige Licht nicht umsonst - es kann ihren Tr�ger in einen Abgrund aus Dunkelheit und Verzweiflung st�rzen lassen.;;;;;;;;;x
```

Comment:
French loc still have ; in the middle of sentence

Reply:
Fixed!